packages/eslint-plugin-vue/src/configs/recommended/recommended.config.ts

```diff
@@ -0,0 +1,35 @@
+import { Linter } from 'eslint';
+
+export const RecommendedConfig: Linter.BaseConfig = {
+    extends: [
+        'plugin:@cloudflight/typescript/recommended',
+        'plugin:vue/vue3-recommended',
+        '@vue/eslint-config-typescript',
+        '@vue/eslint-config-prettier',
+    ],
+    rules: {
+        'vue/block-lang': [
+            'error',
+            {
+                script: {
+                    lang: 'ts',
+                },
+                style: {
+                    lang: 'scss',
+                },
+            },
+        ],
+        'vue/component-api-style': ['error', ['script-setup', 'composition']],
```

Comment:
do we really want to enforce the composition-api? The Vue-docs write that the options-api is more beginner friendly: https://vuejs.org/guide/introduction.html#api-styles

Reply:
I would say yes. The options api does not scale well if your component has more complexity and becomes quite unreadable. I also find the composition api **with** script-setup to have a much better UX.